README.md

```diff
@@ -24,47 +23,49 @@ Kinto-based systems.
 
 ---
 
-## Installation
+## Prerequisites
 
 NodeJS v4+ and npm 2.14+ should be installed and available on your machine.
+## Installation
 
-```bash
-$ npm install -g kinto-admin
-```
+## Installation
 
-Or if you want it to be installed as a project dependency:
+The easiest way to get started is to install [create-react-app](https://github.com/facebookincubator/create-react-app) first:
 
-```bash
-$ npm install kinto-admin --save
+```
+$ npm install -g create-react-app
+$ create-react-app test-kinto-admin && cd test-kinto-admin
+$ npm install kinto-admin --save-dev
 ```
 
-## Standalone local server
+Then, import and render the main `KintoAdmin` component in the generated `src/index.js` file:
 
-The Web admin UI can be served locally using the `kinto-admin` executable:
+```jsx
+import React from "react";
+import ReactDOM from "react-dom";
 
-```bash
-$ kinto-admin serve
-Listening at http://0.0.0.0:3000
-```
+import KintoAdmin from "kinto-admin";
 
-You can specify the port to listen to using the `--port` or `-p` options:
+ReactDOM.render(
+  <KintoAdmin />,
+  document.getElementById("root")
+);
 
-```bash
-$ kinto-admin serve -p 4000
-Listening at http://0.0.0.0:4000
 ```
 
-## Building static assets for production hosting
+To run a local development server:
 
 ```
-$ kinto-admin build -d static-build
+$ npm start
 ```
 
-Here, production-ready assets are generated into the `static-build/` directory,
-and can be published to any static webserver like
-[github-pages](https://pages.github.com/).
+To build the admin as a collection of static assets, ready to be hosted on a static webserver:
+
+```
+$ npm run build
```

Comment:
Users will deal with this when building their own configured admin using create-react-app.